modules/dfpAdServerVideo.js

```diff
@@ -85,6 +114,32 @@ export default function buildDfpVideoUrl(options) {
   });
 }
 
+/**
+ * Builds a video url from a base dfp video url and a winning bid, appending
+ * Prebid-specific key-values.
+ * @param {Object} components base video adserver url parsed into components object
+ * @param {Object} bid winning bid object to append parameters from
+ * @return {string} video url
+ */
+function buildUrlFromAdserverUrlComponents(components, bid) {
+  if (!config.getConfig('usePrebidCache')) {
+    if (!deepAccess(components, 'search.description_url')) {
+      const vastUrl = bid && bid.vastUrl;
+      components.search.description_url = encodeURIComponent(vastUrl);
+    } else {
+      logError(`input url cannnot contain description_url`);
+    }
+  }
```

Comment:
This `description_url` validation block is basically copy/paste between `buildDfpVideoUrl` and `buildUrlFromAdserverUrlComponents`... would be more maintainable in a helper method.